src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -49,12 +50,34 @@ object VertexSchema {
   }
 }
 
+object EdgeSchema {
+  val sep = "_"
+  sealed trait EdgeRelationType {
+    def prefix: String
+    def toLabel(label:String) = prefix + sep + label
```

Comment:
Minor point, but it's confusing to have a ```toLabel``` function that takes ```label```. Especially since ```label``` is also used below to denote the whole thing. Maybe ```suffix``` instead?